!BoundsCheckException
	in X.foo() [opt_xbnd_for04.st.v3 @ 7:26]
	in m__n() [opt_xbnd_for04.st.v3 @ 13:27]
	in main() [opt_xbnd_for04.st.v3 @ 1:18]

